src/Resources/contao/dca/tl_content.php

```diff
@@ -94,7 +94,7 @@
     'options' => array(
         'bullets' => 'Punkte',
         'fraction' => 'Bruchzahl',
-        'progress' => 'Fortschritt'
+        'progressbar' => 'Fortschritt'
```

Comment:
Can you please change this back? This would create a BC break.

Reply:
I have undone the changes and added a query in `WrapperStop.php` so that *progress* turns into *progressbar* as this is the correct value in swiper.js.